lib/Crypto/lockcoderequest.h

```diff
@@ -44,6 +46,13 @@ class SAILFISH_CRYPTO_API LockCodeRequest : public Sailfish::Crypto::Request
     };
     Q_ENUM(LockCodeTargetType)
 
+    enum LockStatus {
+        Unknown = 0,
```

Comment:
Mentioned already in irc but I do think that Unsupported or such would be good here if extension plugin does not support locking.

Reply:
Yep, will add that, thanks